docs/intro/installation.md

```diff
@@ -259,12 +259,66 @@ agave-install update
 programs. The Anchor framework leverages Rust macros to simplify the process of
 writing Solana programs.
 
-Install the Anchor CLI with the following command:
+There are two ways to install the Anchor CLI and tooling:
+
+1. Using Anchor Version Manager (AVM) - the is the **recommended installation**
+   method since it simplifies updating Anchor versions in the future
+2. Without AVM - this requires more a manual process to update Anchor versions
+   later
+
+<Tabs groupId="anchor" items="AVM, Without AVM">
+<Tab value="AVM">
+
+The Anchor version manager (AVM) allows you to install and manage different
+Anchor versions on your system, including more easily updating Anchor versions
+in the future.
+
+Install AVM with the following command:
+
+```shell
+cargo install --git https://github.com/coral-xyz/anchor avm --force
+```
+
+Test to ensure AVM was installed and is accessible:
+
+```shell
+avm --version
+```
+
+Install the latest version of Anchor CLI using AVM:
+
+```shell
+avm install latest
+avm use latest
+```
+
+Or install a specific version of the Anchor CLI by declaring which version you
+want to install:
+
+```shell
+avm install 0.31.0
+avm use 0.31.0
```

Comment:
It doesn't look like 0.31.0 has released, was it intentional to have it here?

Reply:
oooh good catch. just a fat finger typo on my part. I meant to put `0.30.1`

Reply:
now fixed!